client/src/app/(modules)/network/(form)/new/project/page.tsx

```diff
@@ -0,0 +1,529 @@
+      zod: z
+        .string()
+        .max(255, {
+          message: 'Name is limited to 255 characters.',
+        })
+        .optional(),
+      type: 'text',
+      maxSize: 255,
+    },
+    second_project_coordinator_email: {
+      label: 'Email',
+      zod: z
+        .string()
+        .email('Please, enter a valid email.')
+        .max(255, {
+          message: 'Email is limited to 255 characters.',
+        })
+        .optional(),
+      type: 'email',
+    },
+    name: {
+      label: 'Name',
+      required: true,
+      zod: z.string().nonempty('Field is required').max(255, {
+        message: 'Name is limited to 255 characters.',
+      }),
+      type: 'text',
+      maxSize: 255,
+    },
+    website: {
+      label: 'Website',
+      required: true,
+      zod: z
+        .string()
+        .max(255, {
+          message: 'Website is limited to 255 characters.',
+        })
+        .regex(new RegExp('^(https?:\\/\\/)?(www\\.)?[a-zA-Z0-9-]+(\\.[a-zA-Z0-9-]+)+$'), {
+          message: 'Please, enter a valid URL.',
+        }),
+      type: 'text',
+      maxSize: 255,
+      description: (
+        <div className="leading-normal text-gray-500">
+          Accepted URLs format:
+          <ul className="ml-4 list-disc">
+            <li>https://irc-orcasa.eu/ or https://www.irc-orcasa.eu/</li>
+            <li>www.irc-orcasa.eu/</li>
+            <li>irc-orcasa.eu/</li>
+          </ul>
+        </div>
+      ),
+    },
+    short_description: {
+      label: 'Brief description',
+      required: true,
+      zod: z
+        .string()
+        .max(350, {
+          message: 'Brief description is limited to 350 characters.',
+        })
+        .nonempty('Field is required'),
+      type: 'textarea',
+      maxSize: 350,
+    },
+    description: {
+      label: 'Description',
+      zod: z
+        .string()
+        .max(3000, {
+          message: 'Description is limited to 3000 characters.',
+        })
+        .optional(),
+      type: 'textarea',
+      maxSize: 3000,
+    },
+    project_type: {
+      label: 'Project type',
+      zod: z
+        .enum(projectTypes?.map((type) => type.id.toString()) as [string, ...string[]])
+        .optional(),
+      type: 'select',
+      options: projectTypes?.map((type) => ({
+        label: type.name,
+        value: type.id.toString(),
+      })),
+    },
+    start_date: {
+      label: 'Start date',
+      required: true,
+      zod: z.string(),
+      type: 'date',
+    },
+    end_date: {
+      label: 'End date',
+      zod: z.string().optional(),
+      type: 'date',
+    },
+    country_of_coordination: {
+      label: 'Country of coordination',
+      zod: z
+        .enum(countries?.map((type) => type?.id?.toString()) as [string, ...string[]])
+        .optional(),
+      type: 'select',
+      options: countries?.map((country) => ({
+        label: country?.name,
+        value: country?.id?.toString(),
+      })),
+    },
+    regions_of_intervention: {
```

Comment:
When submitted, this field is not saved in Strapi. I wonder if it's because the name is different there: `region_of_interventions`.